### DISCUSSION TITLE
Delay in Web interface responses when a node is down or stopped

### DISCUSSION THREAD

**Discussion by pavel-semenchukov** [rabbitmq/rabbitmq-server#10541] [Questions] [answered]
Good evening, I have a problem. I am configuring a cluster of 3 nodes. Everything works as I want, except for one moment, if at least one node stops working, I have very big delays when using the web interface, switching to Queues and Streams or Exchanges or Overview tabs happens with a very big delay, 20-30 seconds. Can you tell me who has encountered this, how did you manage to solve it? I have tried many different parameters in the config, but I was not able to change this behavior

**Comment by lukebakken** [comment]
We can't assist you without more information:
* RabbitMQ version
* Erlang version
* Operating system running RabbitMQ, and version

**Reply by pavel-semenchukov** [reply]
I have 
RabbitMQ 3.12.12
Erlang 26.2.1
Operating System: Ubuntu 22.04.3 LTS

**Reply by pavel-semenchukov** [reply]
I'm currently setting up the cluster for training purposes, and there is no real client load on it.

**Comment by michaelklishin** [comment, marked as answer]
This has been discussed several times fairly recently:

 * https://github.com/rabbitmq/rabbitmq-server/discussions/10100
 * https://github.com/rabbitmq/rabbitmq-server/discussions/9522
 * https://github.com/rabbitmq/rabbitmq-server/pull/9874 (will ship in 3.13.0)
 * https://github.com/rabbitmq/rabbitmq-server/pull/10065 (will ship in 4.0)

There isn't much else to add, really.
